change/@fluentui-react-combobox-1d78745f-864b-42ce-b821-16fbc1be84dc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32287&commitId=8fe45cbba6ed94585e680bd5a1a8a60647f84060&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|7| Changed|
|Avatar Converged.badgeMask.chromium.png|11| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1139| Changed|
|Drawer.Full Overlay RTL.chromium.png|2890| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>